schema/me/follow_artist.js

```diff
@@ -2,6 +2,7 @@ import gravity from "lib/loaders/legacy/gravity"
 import { GraphQLString, GraphQLBoolean } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 import { ArtistType } from "schema/artist/index"
+import PopularArtistsType from "schema/artists/popular"
```

Comment:
since we're doing `export default PopularArtists` from the schema, you _can_ call it anything you'd like here, so this still works, but its a bit confusing.

Mind renaming this import line to `import PopularArtists from ...` , and then below update to `popular_artists: PopularArtists`.